build_projects/dotnet-cli-build/build.ps1

```diff
@@ -71,7 +71,7 @@ $env:DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 
 # Restore the build scripts
 Write-Host "Restoring Build Script projects..."
-pushd "$PSScriptRoot\.."
+pushd "$PSScriptRoot"
```

Comment:
There are none anymore. I changed it to all go in 1 assembly cuz MSBuild can't load dependencies strong next to this assembly.